pug/pages/todo.pug

```diff
@@ -42,6 +42,13 @@ block content
 					'title':'Room allotment',
 					'desc' :'This would take place in MMCA office at KBH for Boys and at AGH for girls. Rooms will be allotted on random basis or choice upon request'
 				},
+				{
+					'title':'Optional-After the room allotment',
+					'desc' :'Welcome to NIT-Hamirpur! Now that all the tiresome admission process is over, there are still a few days for the classes to begin.
+					         You can explore the place where you'll be spending 4-5 years of your life <a href="http://nith.ac.in/" target="_blank">here</a> or take a look at the <a href="http://nith.ac.in/wp-content/uploads/2018/08/ug_manual_oct15.pdf" target="_blank">UG Manual</a> .
```

Comment:
```suggestion
					         You can explore the place where you\'ll be spending 4-5 years of your life <a href="http://nith.ac.in/" target="_blank">here</a> or take a look at the <a href="http://nith.ac.in/wp-content/uploads/2018/08/ug_manual_oct15.pdf" target="_blank">UG Manual</a>.' +
```
Escape any single quotation mark that occurs in middle of sentence with \ in front of it